README.md

```diff
@@ -12,6 +12,16 @@
 
 ## 🛠️ Installation
 
+### Automatic Installation
+
+```bash
+curl -fsSL https://rafay99.com/snaprescue.sh | sh
+```
+
+- Copy this command and paste it in your terminal.
```

Comment:
change it to: Copy the above command, paste in your terminal and hit ENTER.

Reply:
sure, this is a nice change